.gitignore

```diff
@@ -9,12 +9,7 @@
 *.log
 
 # autogenerated chain_specs
+chain_dumps/
```

Comment:
I suggest to call this `spec_dumps` and you need to add the empty directory too or this will fail

Reply:
I don't quite agree, it is not only the chain_specs. The `.state` and `.wasm` files are also contained.

What do you mean by 'adding the empty directory'? 
The dump script runs `mkdir -p` to make sure the directory exists, I don't think we need to check this in.

Reply:
missed that. sorry. can live with chain